extensions/pomodoro/src/pomodoro-control-timer.tsx

```diff
@@ -107,8 +107,29 @@ const ActionsList = () => {
   );
 };
 
+const handleQuote = (): string => {
+  let markdownContent = "";
+  const quoteUrl = "https://api.quotable.io/quotes/random?limit=1";
+  // Get Quote here
+  const backupQuote = { content: "You did it!", author: "Unknown" }; // Backup quote if the API returns no data
+  const { isLoading, data } = useFetch(quoteUrl, { keepPreviousData: true }) as {
+    isLoading: boolean;
+    data: QuoteResponse;
+  };
+  if (!isLoading) {
+    markdownContent += data.length ? formatQuote(data[0]) : formatQuote(backupQuote);
+  }
+
+  return markdownContent;
+
+  function formatQuote(quote: { content: string; author: string }) {
+    // Format the quote in markdown:
+    return `> ${quote.content} \n>\n> &dash; ${quote.author}`;
+  }
+};
+
 const EndOfInterval = () => {
-  let markdownImage;
+  let markdownContent = "# Timer Completed \n\n";
```

Comment:
Perhaps would be nice to keep the same language. Instead of Timer perhaps using Interval?
What do you think?

Reply:
Sounds good, I'll update this now